File: WebUI/src/app/models/address.ts
export class Address {
    public line1: string;
    public line2: string;
    public city: string;
    public country: string;
    public zipCode: string;

    constructor(line1: string, line2: string, city: string, country: string, zipCode: string) {
        this.line1 = line1;
        this.line2 = line2;
        this.city = city;
        this.country = country;
        this.zipCode = zipCode;
    }
}

File: backend/PartyFindsApi/Models/Address.cs
// <copyright company="PartyFinds LLC">
//   Copyright (c) PartyFinds LLC.  All rights reserved
// </copyright>

using Newtonsoft.Json;

namespace PartyFindsApi.Models
{
    [JsonObject]
    public class Address
    {
        [JsonProperty("line1", Required = Required.Always)]
        public string Line1 { get; set; }

        [JsonProperty("line2")]
        public string Line2 { get; set; }

        [JsonProperty("city", Required = Required.Always)]
        public string City { get; set; }

        [JsonProperty("country", Required = Required.Always)]
        public string Country { get; set; }

        [JsonProperty("zipCode", Required = Required.Always)]
        public string ZipCode { get; set; }

        [JsonProperty("latitude")]
        public string Latitude { get; set; }

        [JsonProperty("longitude")]
        public string Longitude { get; set; }
    }
}
